content/2.0.html.haml

```diff
@@ -0,0 +1,176 @@
+---
+layout: default
+title: Jenkins 2.0
+---
+
+
+:css
+  #download DT {
+    font-size: 1.2em;
+    font-weight: bold;
+    margin: 0.2em 0;
+    padding: 0;
+  }
+  #download DD IMG {
+    vertical-align: middle;
+  }
+  #download DL {
+    margin: 0;
+  }
+  #highlights DT {
+    font-weight: bold;
+    margin-top: 1em;
+  }
+  #highlights DD {
+    padding-left: 1em;
+    margin-bottom: 1em;
+  }
+  #previewdownloads {
+    float: right;
+    width: 200px;
+    padding: 1em;
+    background: #d3d7cf;
+    margin-left: 1em;
+  }
+
+
+%h1
+  Jenkins 2.0 preview
+
+%p
+  %strong
+    Jenkins 2.0
+  is an upcoming major version of Jenkins with a number of usability enhancements and features.
+
+%div#previewdownloads
+  %h2
+    Preview binaries
+
+
+  %dl#download
+    %dt
+      Download the .war
+
+    %dd
+      %div
+        %a{:href => 'http://mirrors.jenkins-ci.org/war-rc/2.0/jenkins.war'}
+          jenkins.war
+
+    %dt
+      Download native packages
+
+    %dd.release-block-platforms
+      %div
+        %img{:src => "/images/os/win_other.png"}
+          %a.release-block-soft{:href => "http://mirrors.jenkins-ci.org/windows-rc/jenkins-2.0.zip"}
+            Windows
+      %div
+        %img{:src => "/images/os/ubuntu.png"}
+          %a.release-block-soft{:href => "http://pkg.jenkins-ci.org/debian-rc/jenkins_2.0_all.deb/"}
+            Ubuntu/Debian
+      %div
+        %img{:src => "/images/os/redhat.png"}
+          %a.release-block-soft{:href => "http://pkg.jenkins-ci.org/redhat-rc/jenkins-2.0-1.1.noarch.rpm/"}
+            Red Hat/Fedora
+      %div
+        %img{:src => "/images/os/os_macosx.png"}
+          %a.release-block-soft{:href => "http://mirrors.jenkins-ci.org/osx-rc/jenkins-2.0.pkg"}
+            Mac OS X
+      %div
+        %img{:src => "/images/os/opensuse.png"}
+          %a.release-block-soft{:href => "http://pkg.jenkins-ci.org/opensuse-rc/jenkins-2.0-1.2.noarch.rpm"}
+            openSUSE
+
+
+%h2
+  Highlights
+
+%dl#highlights
+  %dt
+    %strong
+      Pipeline as Code
+    keeps delivery
+    %a{:href => 'https://en.wikipedia.org/wiki/Don%27t_repeat_yourself'}
+      DRY
+  %dd
+    %p
+      The new
+      %a{:href => '/solutions/pipeline'}
+        Pipeline
+      functionality in Jenkins allows you to define configuration as code,
+      which can be checked in and version controlled along with the rest of
+      your project's source code.
+
+    %img{:src => '/images/hello-world-pipeline.png', :title => 'A simple build/test pipeline'}/
+
+    %p
+      Defining your pipeline's configuration as code makes it easier to build a
+      very simple "build and test" pipeline, while enabling more advanced and
```

Comment:
Perhaps reduce duplication here:
`easier to build a very simple "build and test" pipeline"` -> `easier to create a very simple...`, or `easier to set up a very simple...`